test-requirements.txt

```diff
@@ -0,0 +1,2 @@
+odoo14-addon-product-packaging-type @ git+https://github.com/trobz/product-attribute@14.0-mig-product_packaging_type#subdirectory=setup/product_packaging_type
+odoo14-addon-product-packaging-dimension @ git+https://github.com/dzungtran89/product-attribute@14.0-mig-product_packaging_dimension#subdirectory=setup/product_packaging_dimension
```

Comment:
@dzungtran89 Could you remove this as already merged ?

Reply:
sure, I updated it, including a fix regarding to a new change on https://github.com/OCA/product-attribute/pull/850